ocs_ci/resiliency/platform_stress.py

```diff
@@ -0,0 +1,298 @@
+import time
+import subprocess
+import random
+import threading
+import logging
+from ocs_ci.ocs.exceptions import (
+    CommandFailed,
+    NoRunningCephToolBoxException,
+)
+from ocs_ci.ocs import ocp
+
+log = logging.getLogger(__name__)
+
+
+class PlatformStress:
```

Comment:
@paraggit @PrasadDesala @mashetty330 please coordinate this with the stress automation